src/io/flutter/FlutterBundle.properties

```diff
@@ -123,15 +123,30 @@ flutter.module.create.settings.help.project_type.label=Project type:
 flutter.module.create.settings.help.project_type.description.app=Select an "Application" when building for end users.
 flutter.module.create.settings.help.project_type.description.plugin=Select a "Plugin" when exposing an Android or iOS API for developers.
 flutter.module.create.settings.help.project_type.description.package=Select a "Package" when creating a pure Dart component, like a new Widget.
+flutter.module.create.settings.help.type.application=An "application" is a stand-alone app that can be installed and run on a mobile device.
```

Comment:
I would suggest removing "mobile" to make it more general. Maybe a device means it's a chrome device that runs android.

Reply:
Good point. I noticed that the flutter.io site has "mobile device" a few times, but "device" is used far more often.